content/20.pharmaceuticals.md

```diff
@@ -584,6 +595,23 @@ Computational screening allows for a large number of compounds to be evaluated t
+This pattern applied only to first-in-class drugs, with most follower drugs produced by target-based screening [@doi:10.1038/clpt.2012.236].
+These findings suggest that target-based drug discovery is more successful when building on a known MMOA, and that modulating a target is most valuable when the target is part of a valuable MMOA [@doi:10.1038/nrd4366].
+Building on this, many within the field suggested that mechanism-informed phenotypic investigations may be the most useful approach to drug discovery [@doi:10.1038/nrd4366; @doi:10.1016/j.chembiol.2015.11.008; @doi:10.1126/scitranslmed.aab1201].
+This shift highlights the importance of prioritizing candidates that are likely to have a desired effect beyond computational or experimental models.
+While this is easier said than done, efforts to fill in the black box underlying computational HTS and recognize patterns among the identified compounds aid in moving data-oriented drug repurposing efforts in this direction.
+Data-driven efforts to identify patterns in the results of computational screens allowed researchers to notice the shared property of cationic amphicility among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].
+
+The strategies outlined here for drug repurposing reflect the same patterns found in HTS.
+In COVID-19, candidates for repurposing have been identified through both phenotypic (clinically indicated) and target-based approaches.
+Drugs like dexamethasone have a known effect on the human body, and in the case of COVID-19, this effect was found to be beneficial to patient outcomes.
+Attempts to target the virus or host-virus interactions have produced mixed results in the clinic.
+Remdesivir, which has emerged as a success for the target-based approach to identifying candidate COVID-19 therapeutics for repurposing, was first identified in a hypothesis-driven HTS effort to find compounds active against RNA viruses _in vitro_ [@doi:10.1021/acs.jmedchem.6b01594].
+Promising preclinical results translated poorly to an initial RCT for Ebola [@doi:10.7573/dic.2020-4-14], although the results have been more promising for COVID-19 (Table @tbl:summary-table).
+Favipiravir, which uses a similar mechanism, has not been supported as a therapeutic for COVID-19, and HCQ, which was proposed to have antiviral effects relevant to the treatment of COVID-19 based on a mechanism that is known to translate poorly _in vivo_, was found to be harmful to patients.
+The unpredictable nature of successes and failures of mechanism-based drug repurposing thus highlights one of the tenets of phenotypic screening: there are a lot of "unknown unknowns", and a promising mechanism at the level of an MMOA will not necessarily propogate up to the pathway, cellular, or organismal level [@doi:10.1126/scitranslmed.aab1201].
```

Comment:
"mechanism-based drug repurposing" is yet another new term that could confuse the existing classifications described.

Reply:
```suggestion
The unpredictable nature of successes and failures of drug repurposing thus highlights one of the tenets of phenotypic screening: there are a lot of "unknown unknowns", and a promising mechanism at the level of an MMOA will not necessarily propagate up to the pathway, cellular, or organismal level [@doi:10.1126/scitranslmed.aab1201].
```